### PR TITLE
Enh/checkbox choices

### PR DESCRIPTION
**Pull request by sheralim012**

### CODE REVIEW

File: questionnaires/models.py [hunk 1]
```diff
@@ -620,7 +615,7 @@ def choice_index(self, choice):
             if self.field_type == 'checkbox':
                 # clean checkboxes have True/False
                 try:
-                    return ['on', 'off'].index(choice)
+                    return ['true', 'false'].index(choice)
```
Comment (ehmadzubair) [inline questionnaires/models.py:614]: remove this code

File: questionnaires/static/js/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        }
+        fieldType.change(e => {
+            if (['hidden', ''].includes(fieldType.val())) {
+                choicesField.css('display', 'none');
+            } else {
+                choicesField.css('display', 'block');
+            }
+            if (e.target.value === 'checkbox') {
+                question.find('[id$=-choices]').first().val('true|false');
+            } else {
```
Comment (ehmadzubair) [inline questionnaires/static/js/index.js]: remove this else

**Comment by ehmadzubair** [comment]
👍🏼

File: questionnaires/models.py (hunk 1 above)
Comment (sheralim012) [reply questionnaires/models.py:614]: done

File: questionnaires/static/js/index.js (hunk 2 above)
Comment (sheralim012) [reply questionnaires/static/js/index.js]: done

**Pull request closed without merging**